### ISSUE TITLE
Give a link to BMO user-profile page

### ISSUE THREAD

**Issue by debloper** [MozillaIndia/leaderboard#36]
The https://bugzilla.mozilla.org/user_profile?login=username%40email.tld can provide some more\* useful insights about one's BMO activities.

> Especially if someone is involved more with locked/private bugs.

**Comment by grssam** [comment]
They will have to login to bugzilla to get that info. and that info is not as much detailed as we can get. Specially when we are aiming for BZ + GH + other details too.
The link can still be present, but this issue will not make #35 invalid.

**Comment by debloper** [comment]
Yeah, it doesn't make #35 invalid; especially if we continue with multiple channels for inbound data (gh, bz etc.)
However, one doesn't need to have been logged in to use the BMO user-profiles. Example:
- https://bugzilla.mozilla.org/user_profile?login=sankha93%40gmail.com
- https://bugzilla.mozilla.org/user_profile?login=scrapmachines%40gmail.com

**Comment by grssam** [comment]
Okay, yeah, it works without logging in :) (my bad)

On Mon, Aug 26, 2013 at 5:57 PM, Soumya Deb notifications@github.comwrote:

> Yeah, it doesn't make #35https://github.com/MozillaIndia/leaderboard/issues/35invalid; especially if we continue with multiple channels for inbound data
> (gh, bz etc.)
> However, one doesn't need to have been logged in to use the BMO
> user-profiles. Example:
> - https://bugzilla.mozilla.org/user_profile?login=sankha93%40gmail.com
>   -
>   https://bugzilla.mozilla.org/user_profile?login=scrapmachines%40gmail.com
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/MozillaIndia/leaderboard/issues/36#issuecomment-23259028
> .

## 

Girish Sharma
B.Tech(H), Civil Engineering,
Indian Institute of Technology, Kharagpur
